src/tree/xpath/XPath.ts

```diff
@@ -147,8 +142,8 @@ export class XPath {
 	}
 
 	/**
-	 * Convert word like {@code *} or {@code ID} or {@code expr} to a path
-	 * element. {@code anywhere} is {@code true} if {@code //} precedes the
+	 * Convert word like `*` or `ID` or `expr` to a path
```

Comment:
I'm pretty ignorant about JSDoc, but I remember reading in the docs that JSDoc has a Markdown parser as an add-in, but that by default, it [only looks for markdown in specific JSDoc tags.](http://usejsdoc.org/plugins-markdown.html).   

I like the idea of markdown better than tags like @code, but I don't see that this is in any of the named tags.    I don't know if this is a problem, do you know more than I, or someone else who might?

Reply:
I just just going based on what Quick Info (hover tips) looks like in VS Code. It's hard to tell what's right and what's wrong.